ClinGen/clincoded
# Connects to #401. Connects to #526. Spell out methods and fix js for IE

Testing for #401:

1. Create GDM
2. Add PMID
3. Create Group, Family, Individual objects with new Sanger and HRM method items and confirm that they are viewable and editable
4. Test instance w/ live data to confirm that upgrade scripts work properly.

Testing for #526:

1. Confirm login works in IE
